src/schema/v2/me/myCollectionDeleteArtworkImageMutation.ts

```diff
@@ -0,0 +1,59 @@
+import { GraphQLString, GraphQLNonNull } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { ResolverContext } from "types/graphql"
+import { MyCollectionArtworkMutationType } from "./myCollection"
+import { formatGravityError } from "lib/gravityErrorHandler"
+
+export const MyCollectionDeleteArtworkImageMutation = mutationWithClientMutationId<
+  any,
+  any,
```

Comment:
non-blocking: A couple more examples of `any` that might be worth digging into what types they _could_ be, but I'm adventured out and not going to press hard on this.

Reply:
The first of these corresponds to the inputs. So this line: https://github.com/artsy/metaphysics/pull/2879/files?file-filters%5B%5D=.ts#diff-adc1cc7caf33742997c144736e795bdce900110ac5a7cf76023676bc86603865R29 likely has `artworkId` and `imageId` as `any`. Can update the typing here to `{ artworkId: strinng, imageId: string }`

The second of these is the type of `result` in https://github.com/artsy/metaphysics/pull/2879/files?file-filters%5B%5D=.ts#diff-adc1cc7caf33742997c144736e795bdce900110ac5a7cf76023676bc86603865R25 (can prob be fine as `any`).